README.md

```diff
@@ -54,16 +54,17 @@ To tun the tests:
 pytest pythonpro
 ```
 
-If you want run your amb dev using postgres, you can install docker and run:
+If you want run your amb dev using postgres, you can add to your .env 
 
 ```console
-docker-compose docker-compose.yml up 
+DATABASE_URL=postgres://postgres:pass@localhost:5432/postgres
 ```
 
-and add to your .env
+
+and install docker and run:
 
 ```console
-DATABASE_URL=postgres://postgres:pass@localhost:5432/postgres
+docker-compose -f docker-compose.yml up 
```

Comment:
What is the effect of using "-f" option

Reply:
In fact, docker-compose.yml is de default file. So please update this command to only "docker-compose up"

Reply:
oh great, I make change.